### DISCUSSION TITLE
Google購買核電，開創先例，台灣是否該跟進核電？

### DISCUSSION THREAD

**Discussion by JaYanlloz** [1101-GS4719-C/Python-Community-GS4719-C#2170] [科技新知]
Google最近跟能源新創Kairos Power簽了購電協議，成為了全球第一家買核電的科技公司。他們打算從2030年起，購買7座小型模組化核反應爐(SMR)所產生的電力，總共增加500MW的核能，這些反應爐將在2030年前上線，2035年前還要擴大規模。我認為這動作會帶動美國核能商業化，也讓核能在科技產業的應用走在向前邁出一大步

看了輿論風向，大家都認為SMR是核電產業的未來
這不免讓人開始思考，面對日益增長的能源需求，台灣是否也該考慮跟進，重新審視核電？

**Comment by 112121py** [comment]
也許可以考慮，但目前應該還不太行，問題就跟傳統核電廠一樣，雖然SMR的安全性、汙染等等都比傳統的還要少，但依然還是有核廢料等等的問題，想要產生跟傳統一樣的電力，需要設置非常多才能達成，還有零件的置換。要設置在哪裡，廢料又要放在誰家附近，民眾的意見想必也是需要妥善處理跟考量的一環，但如果有好好處理，相信也能改善現今的核電環境
